src/binder/clrprivbindercoreclr.cpp

```diff
@@ -159,8 +159,13 @@ HRESULT CLRPrivBinderCoreCLR::BindUsingPEImage( /* in */ PEImage *pPEImage,
                 {
                     if (pCoreCLRFoundAssembly->GetIsInGAC())
                     {
-                        // If we were able to bind to a TPA assembly, then fail the load
```

Comment:
We may want to delete or move the `IsMscorlib` case above - so that this kicks for it as well.